js-intro.js

```diff
@@ -23,17 +40,39 @@ var myCat = "Garfield"
 
 // 2a. Create a function that returns the longer of the two variables. Expected output: "Garfield"
 
-// 2b. Create a function that takes in two arguments and returns an array containing the two variables. Expected output: ["Snoopy", "Garfield"]
+function longerName(){
+    if (myDog.length > myCat.length) {
+         return myDog
+    } else if
+        (myCat.length > myDog.length)
+         return myCat
+}
+console.log(longerName())
 
```

Comment:
The if/else block here looks great, but your function isn't really doing anything. If you add arguments/placeholders to your code, this will be perfect.